CMake/External_OpenCV.cmake

```diff
@@ -197,6 +222,7 @@ ExternalProject_Add(OpenCV
     -DBUILD_SHARED_LIBS:BOOL=True
     -DBUILD_TESTS:BOOL=False
     -DWITH_EIGEN:BOOL=${fletch_ENABLE_EIGEN}
+    -DWITH_JASPER:BOOL=False
```

Comment:
Is there a good reason to ever leave jasper on? What is it about CUDA that messes with the Jasper components?